apps/new-dealer/views/partials/company.html

```diff
@@ -0,0 +1,6 @@
+<div id="company-name-panel" class="reveal" aria-hidden="false">
+  <div class="panel-indent">
+    {{#input-text}}company-name{{/input-text}}
```

Comment:
otherwise there are 2 places this field is tied to type - field config and the template